dpnp/dpnp_iface_manipulation.py

```diff
@@ -1332,6 +1333,82 @@ def swapaxes(a, axis1, axis2):
     )
 
 
+def tile(A, reps):
+    """
+    Construct an array by repeating `A` the number of times given by reps.
+
+    If `reps` has length ``d``, the result will have dimension of
+    ``max(d, A.ndim)``.
+
+    If ``A.ndim < d``, `A` is promoted to be d-dimensional by prepending new
+    axes. So a shape (3,) array is promoted to (1, 3) for 2-D replication,
+    or shape (1, 1, 3) for 3-D replication. If this is not the desired
+    behavior, promote `A` to d-dimensions manually before calling this
+    function.
+
+    If ``A.ndim > d``, `reps` is promoted to `A`.ndim by prepending 1's to it.
+    Thus for an `A` of shape (2, 3, 4, 5), a `reps` of (2, 2) is treated as
+    (1, 1, 2, 2).
+
+    Note : Although tile may be used for broadcasting, it is strongly
+    recommended to use numpy's broadcasting operations and functions.
+
+    Parameters
+    ----------
+    A : dpnp_array
+        The input array.
+    reps : array_like
+        The number of repetitions of `A` along each axis.
+
+    Returns
+    -------
+    c : dpnp_array
+        The tiled output array.
+
+    See Also
+    --------
+    :obj:`dpnp.repeat` : Repeat elements of an array.
+    :obj:`dpnp.broadcast_to` : Broadcast an array to a new shape
```

Comment:
probably it would be good to add a link to `numpy.tile` at least in `see also` block, like cupy does